src/dxvk/meson.build

```diff
@@ -98,6 +98,9 @@ dxvk_src = files([
   'dxvk_stats.cpp',
   'dxvk_unbound.cpp',
   'dxvk_util.cpp',
+  'dxvk_extension_provider.cpp',
+
+  'platform/dxvk_win32_exts.cpp',
```

Comment:
I'd rather have this file inside a `dxvk/platform` subdirectory rather than a new top-level one.

Reply:
Actually, forget about that if there's more plaform stuff to come in the future.

Reply:
Extension providers can be anything that could potentially influence the extensions we need. The change also made OpenVR an extension provider. In my mind it's not *just* a platform thing.